src/generated/swagger/configv1/spec.json

```diff
@@ -1539,7 +1598,7 @@
           "type": "string"
         },
         "allocation": {
-          "$ref": "#/definitions/ResourcePoolsAllocation"
+          "$ref": "#/definitions/configv1ResourcePoolsAllocation"
```

Comment:
Is this expected? Between the ref change and many file renames, I can't tell if there's a tooling change or versioning issue.

Reply:
Honestly not sure what that's about. All I did was the swagger update. Maybe someone on programmable platform will be aware

Reply:
I'll let them chime in because I don't really know how this stuff works 😬

Reply:
@bg451 might be the best to say for sure. I can dig in a bit and see what i can find.  IIRC the $ref stuff in swagger spec is just pointing to a type although i wonder if it would cause a different type to be generated in the case of client generation. It changed in this commit a few weeks ago: https://github.com/chronosphereio/monorepo/commit/d340b2cb83bf315e259a8250323e37fe3ad92805#diff-8e6a36bbf7407b7defe60bca8c61ac8e3b8a3ce4feb99981fe594dc42f1affee

Reply:
Our swagger generator was bumped versions about a month ago. Generally, ref renames are safe and are not a breaking change. Additionally, the contract here is around the API itself, not the swagger config or generated code.